Yoast/wordpress-seo
# Add passive voice assessment for Greek

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements passive voice assessment for Greek.
* Adds passive voice assessment for Greek.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Enable the `GREEK_SUPPORT` feature by adding `define( 'YOAST_SEO_GREEK_SUPPORT', true );` in `basic-wordpress-config.php` file
* Set your website to Greek (Ελληνικά).
* Test this functionality in Yoast SEO Free.
* Create a new empty post and add these sentences to your post:
````
Το σπίτι χτίστηκε από τον πατέρα μου. Το φαγητο είναι μαγειρεμένο από την μαμά μου. Τα ρούχα έχουν διπλωθεί και το σπίτι έχει καθαριστεί. 
````
* Check that the passive voice bullet turns red and all sentences are highlighted.
* Delete the previous text and add these sentences to your post:
````
Αυτός ο μπαμπάς είναι φοβερός. Η γάτα μου σκέφτεται τί θα φάει μετά. Έχω γραμμένη την εργασία μου. Σήμερα έχω λύσει πολλά προβλήματα. Το άρθρο έχει να γραφθεί.
````
* Confirm that none of the sentences is recognized as a passive and none is highlighted
* Disable the `GREEK_SUPPORT` feature by adding `define( 'YOAST_SEO_GREEK_SUPPORT', false );` in `basic-wordpress-config.php` file

* Save the changes in the post

* Reload the page

* Confirm that the Passive voice assessment doesn't appear in the metabox


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* Set your website to Greek (Ελληνικά).
* Test this functionality in Yoast SEO Free.
* Create a new empty post and add these sentences to your post:
````
Το σπίτι χτίστηκε από τον πατέρα μου. Το φαγητο είναι μαγειρεμένο από την μαμά μου. Τα ρούχα έχουν διπλωθεί και το σπίτι έχει καθαριστεί. 
````
* Check that the passive voice bullet turns red and all sentences are highlighted.
* Delete the previous text and add these sentences to your post:
````
Αυτός ο μπαμπάς είναι φοβερός. Η γάτα μου σκέφτεται τί θα φάει μετά. Έχω γραμμένη την εργασία μου. Σήμερα έχω λύσει πολλά προβλήματα. Το άρθρο έχει να γραφθεί.
````
* Confirm that none of the sentences is recognized as a passive and none is highlighted

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-1031
